types/json/binary.go

```diff
@@ -202,19 +202,23 @@ func (bj BinaryJSON) marshalTo(buf []byte) ([]byte, error) {
 	return buf, nil
 }
 
-func (bj BinaryJSON) getInt64() int64 {
+// GetInt64 gets the int64 value.
+func (bj BinaryJSON) GetInt64() int64 {
 	return int64(endian.Uint64(bj.Value))
 }
 
-func (bj BinaryJSON) getUint64() uint64 {
+// GetUint64 gets the uint64 value.
+func (bj BinaryJSON) GetUint64() uint64 {
 	return endian.Uint64(bj.Value)
 }
 
-func (bj BinaryJSON) getFloat64() float64 {
-	return math.Float64frombits(bj.getUint64())
+// GetFloat64 gets the float64 value.
+func (bj BinaryJSON) GetFloat64() float64 {
+	return math.Float64frombits(bj.GetUint64())
 }
 
-func (bj BinaryJSON) getString() []byte {
+// GetString gets the string value.
+func (bj BinaryJSON) GetString() []byte {
```

Comment:
Why not return a string?

Reply:
It's more convenient to use []byte internally.